packages/web/hooks/window/use-window-size.ts

```diff
@@ -10,6 +8,21 @@ export interface WindowSize {
   isExtraLargeDesktop: boolean;
 }
 
+/** Should match settings in tailwind.config.js
+ *
+ *  https://tailwindcss.com/docs/responsive-design
+ */
+export const enum Breakpoint {
+  SM = 640,
+  MD = 768,
+  LG = 1024,
+  XLG = 1152,
+  XL = 1280,
+  XLHALF = 1408,
+  XXL = 1536,
+  XXXL = 1792,
```

Comment:
Thoughts on making this lowercase?
```suggestion
  sm = 640,
  md = 768,
  lg = 1024,
  xlg = 1152,
  xl = 1280,
  xlhalf = 1408,
  xxl = 1536,
  xxxl = 1792,
```